geneticalg/examples/run_ga/run_ga.py

```diff
@@ -1,23 +1,39 @@
 import pandas as pd
-from geneticalg import gen_chromosome, calc_fitness_all, mutate
+from geneticalg import (gen_chromosome,
+                        calc_fitness_all,
+                        mutate,
+                        crossover,
+                        list_duplicates,
+                        del_duplicates,
+                        sort_chromosomes)
 
 video_data = pd.read_csv("./data/video_ts_data.csv", index_col=None)
 video_lengths = pd.read_csv("./data/video_lengths.csv").iloc[:, 0].values
-video_list = pd.read_csv("./data/video_list.csv",
-                         index_col=None).iloc[:, 0].values
+video_list = pd.read_csv("./data/video_list.csv", index_col=None).iloc[:, 0].values
 
 # Define Variables
 max_seconds = 120 * 60
 max_attempts = 10
-numGenerations = 20
+num_generations = 20
 popSize = 20
 
 # Generate Empty Chromosomes
 chromosomes = []
+max_chromosome = []
 
 for i in range(popSize):
-    chromosomes.append(gen_chromosome(max_seconds, video_list, video_lengths))
+    chromosomes.append(gen_chromosome(max_seconds, video_lengths))
 
-calc_fitness_all(chromosomes)
 
-mutate(chromosomes[0], max_attempts, video_lengths, max_seconds)
+for gen in range(num_generations):
+    for child in crossover(chromosomes[0], chromosomes[1], max_attempts, video_lengths, max_seconds):
```

Comment:
without comments hard to grasp why only the first two `chromosomes` out of possibly `20` (limit in line 35... why? no comment) would be subject to crossover.  It might especially be useful to document your code in examples to give a better idea of what is going on